var/spack/repos/builtin/packages/netlib-lapack/package.py

```diff
@@ -108,6 +108,7 @@ class NetlibLapack(CMakePackage):
     depends_on("netlib-xblas+fortran+plain_blas", when="+xblas")
     depends_on("python@2.7:", type="test")
 
+    build_system("cmake")
```

Comment:
This shouldn't be required: packages should only have to define this if they have more than one possible build system.

Reply:
I encountered a CI error without this during the extended build test. This change can be reverted as we're no longer testing that package, but regardless, not including this causes an error. If that's not correct behavior, then that's an issue. If you look at the CI run for the commit where I added this, you can see that the package install complained about not having a build system variant for package netlib-lapack

Reply:
Yes this does sound like a problem: ideally we could resolve it in a separate PR.